math-doc/math/scribblings/math-matrix.scrbl

```diff
@@ -364,7 +364,7 @@ Like @racket[list*->matrix] and @racket[matrix*->list], but for vectors.
 As in the first example, Typed Racket often needs help inferring the type @racket[A].
 }
 
-                                                                                  
+
 @;{==================================================================================================}
```

Comment:
The deleted whitespace makes it a little harder to find the new additions.

Reply:
my apologies, my linter did it automatically. Is it worth leaving this change, or would it be better to add the white space back?